float64 motor_angle1
float64 motor_angle2
float64 motor_angle3
float64 motor_angle4
float64 cart_x
float64 cart_y
float64 cart_z

